### PR TITLE
Link presentation

### PR DESCRIPTION
**Pull request by johscheuer**
Link ContainerDays 2019 Presentation and make some small clean ups in the markdown.

### CODE REVIEW

**Comment by johscheuer** [comment]
@maal @maxbischoff someone ?

File: README.md [hunk 1]
```diff
@@ -62,9 +63,7 @@ EOF
 Test your newly created NetworkPolicy:
 
 ```bash
-$ illuminatio clean run
-```
-```
+illuminatio clean run
```
Comment (maxbischoff) [inline README.md]: Should this be `illuminatio run clean`? It was changed in a direct commit on master from that, can you fix it?

File: authors.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 * Maximilian Bischoff <mbischoff@inovex.de>
 
-# Contributors
+## Contributors
```
Comment (maxbischoff) [inline authors.md]: Also change the `Initial Author` to be `h1` and add `Authors` as tilte?

File: README.md (hunk 1 above)
Comment (johscheuer) [reply README.md]: The change was the other way round: https://github.com/inovex/illuminatio/commit/289a8e0a9ee351944793ee7d038c743507597e38 e.g. the command means `clean up all old illuminatio resources and then run the tests` IMHO the order is correct.

File: authors.md (hunk 2 above)
Comment (johscheuer) [reply authors.md]: `Initial Author` is h1 ?

Comment (johscheuer) [reply authors.md]: But I think I know what you mean.

**Pull request merged**